auth-web/src/components/auth/staff/continuation-application/ExtraprovincialRegistrationBc.vue

```diff
@@ -72,32 +72,54 @@
 </template>
 
 <script lang="ts">
-import { Component, Prop, Vue } from 'vue-property-decorator'
-import { ContinuationReviewFilingIF, ContinuationReviewIF } from '@/models/continuation-review'
-import DateUtils from '@/util/date-utils'
+import { ContinuationFilingIF, ContinuationReviewIF } from '@/models/continuation-review'
+import { defineComponent, reactive } from '@vue/composition-api'
+import CommonUtils from '@/util/common-util'
+import moment from 'moment'
 
-@Component({})
-export default class ExtraprovincialRegistrationBc extends Vue {
-  /** Continuation Review object that comes from parent component. */
-  @Prop({ required: true }) readonly continuationReview: ContinuationReviewIF
+export default defineComponent({
+  name: 'ExtraprovincialRegistrationBc',
 
-  get continuationIn (): ContinuationReviewFilingIF {
-    return this.continuationReview?.filing?.continuationIn
-  }
+  props: {
+    review: { type: Object as () => ContinuationReviewIF, required: true },
+    filing: { type: Object as () => ContinuationFilingIF, required: true }
+  },
 
-  get identifier (): string {
-    return this.continuationIn?.business?.identifier
-  }
+  setup (props) {
+    const state = reactive({
+      get continuationIn (): any {
+        return props.filing?.continuationIn
+      },
 
-  get legalName (): string {
-    return this.continuationIn?.business?.legalName
-  }
+      get identifier (): string {
+        return this.continuationIn?.business?.identifier
+      },
+
+      get legalName (): string {
+        return this.continuationIn?.business?.legalName
+      },
 
-  get foundingDate (): string {
-    const date = DateUtils.apiToDate(this.continuationIn?.business?.foundingDate)
-    return DateUtils.dateToPacificDate(date, true)
+      get foundingDate (): string {
+        return strToPacificDate(this.continuationIn?.business?.foundingDate)
+      }
+    })
+
+    /**
+     * Converts a date-time string to a Pacific date string.
+     * @example
+     * Sample input: "2007-01-23T08:00:00.000+00:00".
+     * Sample output: "Jan 23, 2007".
+     */
+    function strToPacificDate (str: string): string {
+      const date = moment.utc(str).toDate()
+      return CommonUtils.formatDisplayDate(date, 'MMM D, YYYY')
```

Comment:
This common utils method generates the following warning:

![image](https://github.com/user-attachments/assets/9280f8bf-47dc-4e12-a2e1-1d3302be21e5)

Re:

![image](https://github.com/user-attachments/assets/ca55335e-6e43-48eb-a19d-1c554427bebc)

I believe it's the locale string that it doesn't like, and that comment about not working in CI is suspicious, too.

However, I tested that all my date-time conversions are working as expected, so 🤷‍♂️

Reply:
Would just use moment tz https://momentjs.com/timezone/

`moment(date).tz('America/Vancouver')`

Reply:
OK, `moment.tz()` works without producing a warning but I needed to import "moment-timezone" library (another 3MB).

What do you think?

Reply:
3MB is the unpacked size

it's fine, probably will replace it all with Luxon anyways

Reply:
![image](https://github.com/user-attachments/assets/0a7ab594-0a1a-4f70-959e-608457f5d816)